README.md

```diff
@@ -1129,6 +1130,10 @@ new RequestBodyParserMiddleware(10 * 1024, 100); // 100 files with 10 KiB each
 
 A non-exhaustive list of third-party middleware can be found at the [`Middleware`](https://github.com/reactphp/http/wiki/Middleware) wiki page. 
 
+#### PSR-15 Middleware
+
+Additionally we support PSR-15 through the [Friends of ReactPHP](https://github.com/friends-of-reactphp) [PSR-15 middleware adapter](https://github.com/friends-of-reactphp/http-middleware-psr15-adapter).
```

Comment:
I'd remove "Additionally" as the reference to the previous section isn't required and probably confusing.

Reply:
@jsor Good one, updated the PR